app/export.go

```diff
@@ -379,15 +379,13 @@ func (a *App) ExportAllPosts(writer io.Writer) *model.AppError {
 func (a *App) buildPostReplies(postId string) (*[]ReplyImportData, *model.AppError) {
 	var replies []ReplyImportData
 
-	result := <-a.Srv.Store.Post().GetRepliesForExport(postId)
+	result, err := a.Srv.Store.Post().GetRepliesForExport(postId)
```

Comment:
nit: can we rename `result` to the more useful `replyPosts`, now that we don't have a generic result container?

Reply:
Of course 👍

Reply:
@lieut-data @hanzei 

Hey, I can't figure out why the commit couldn't be built?

Reply:
Looks like it worked eventually!